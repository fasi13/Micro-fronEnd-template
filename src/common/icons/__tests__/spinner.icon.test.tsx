import { render } from '@testing-library/react';
import React from 'react';
import { SpinnerIcon } from '../index';

test('It should render SpinnerIcon', async () => {
	const size = { width: 512, height: 512 };

	expect(
		render(
			<SpinnerIcon className="" width={size.width} height={size.height} />,
		),
	).toMatchSnapshot();
});
