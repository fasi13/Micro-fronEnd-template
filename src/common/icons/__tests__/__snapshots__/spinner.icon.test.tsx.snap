// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render SpinnerIcon 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        class=" animate-spin"
        height="512"
        viewBox="0 0 512 512"
        width="512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <lineargradient
          gradientTransform="matrix(.9996 0 0 -.9996 -.022 513.71)"
          gradientUnits="userSpaceOnUse"
          id="a"
          x1="0.625"
          x2="511.648"
          y1="257.824"
          y2="257.824"
        >
          <stop
            offset="0"
            stop-color="#00f2fe"
          />
          <stop
            offset="0.021"
            stop-color="#03effe"
          />
          <stop
            offset="0.293"
            stop-color="#24d2fe"
          />
          <stop
            offset="0.554"
            stop-color="#3cbdfe"
          />
          <stop
            offset="0.796"
            stop-color="#4ab0fe"
          />
          <stop
            offset="1"
            stop-color="#4facfe"
          />
        </lineargradient>
        <path
          d="M407.497 103.365c-40.873-40.873-95.217-63.383-153.02-63.383s-112.147 22.51-153.02 63.383l-9.585 9.585h65.66c11.041 0 19.991 8.95 19.991 19.991s-8.95 19.991-19.991 19.991l-71.701-.001c-15.86-.191-30.735-6.515-41.886-17.807-11.112-11.253-17.274-26.172-17.353-42.011V19.991C26.592 8.95 35.542 0 46.583 0s19.991 8.95 19.991 19.991v61.714l6.611-6.611C121.61 26.669 185.994 0 254.477 0S387.345 26.669 435.77 75.094c31.333 31.333 54.094 70.346 65.825 112.823 2.939 10.642-3.307 21.652-13.948 24.591-1.78.492-3.572.727-5.334.727-8.767.001-16.811-5.811-19.258-14.674-9.894-35.825-29.105-68.743-55.558-95.196zM127.759 431.233a218.234 218.234 0 01-14.207-11.531c-8.273-7.314-20.907-6.533-28.217 1.738-7.313 8.272-6.534 20.907 1.737 28.218a258.494 258.494 0 0016.821 13.652 19.901 19.901 0 0011.918 3.954c6.103 0 12.131-2.786 16.055-8.059 6.59-8.859 4.752-21.382-4.107-27.972zM23.096 297.764c11.002-.923 19.173-10.589 18.251-21.591a219.948 219.948 0 01-.763-18.289c0-11.041-8.95-19.991-19.991-19.991S.602 246.843.602 257.884c0 7.207.304 14.484.903 21.628.875 10.434 9.613 18.323 19.898 18.323.561 0 1.126-.023 1.693-.071zm36.514 48.879c-4.545-10.062-16.387-14.536-26.448-9.991-10.062 4.544-14.535 16.386-9.991 26.447a256.362 256.362 0 009.759 19.366c3.659 6.557 10.462 10.25 17.473 10.25 3.298 0 6.642-.818 9.726-2.539 9.641-5.382 13.093-17.559 7.711-27.2a215.6 215.6 0 01-8.23-16.333zM210.335 468.85a215.33 215.33 0 01-17.673-4.645c-10.548-3.258-21.742 2.649-25.002 13.199-3.26 10.549 2.65 21.742 13.198 25.002a256.192 256.192 0 0020.97 5.512 20.07 20.07 0 004.272.462c9.197 0 17.477-6.388 19.513-15.743 2.35-10.789-4.49-21.439-15.278-23.787zm282.882-209.234c-11.012-.995-20.72 7.102-21.72 18.098a216.88 216.88 0 01-2.425 18.124c-1.926 10.871 5.326 21.246 16.197 23.173 1.18.209 2.353.31 3.511.31 9.513 0 17.944-6.815 19.661-16.507a257.549 257.549 0 002.874-21.476c1.001-10.996-7.103-20.721-18.098-21.722zm-21.571 97.166c-9.601-5.451-21.803-2.083-27.254 7.518a217.28 217.28 0 01-9.69 15.504c-6.242 9.106-3.921 21.549 5.186 27.792a19.892 19.892 0 0011.285 3.503c6.37 0 12.633-3.04 16.507-8.69a257.432 257.432 0 0011.484-18.375c5.45-9.601 2.084-21.802-7.518-27.252zm-87.642 75.347a217.02 217.02 0 01-15.196 10.183c-9.425 5.752-12.4 18.055-6.648 27.479 3.768 6.172 10.344 9.579 17.082 9.579 3.55 0 7.145-.946 10.396-2.931a257.062 257.062 0 0018.008-12.067c8.903-6.528 10.829-19.039 4.301-27.943-6.529-8.901-19.037-10.83-27.943-4.3zm-82.853 37.039a216.53 216.53 0 01-18.031 2.998c-10.959 1.347-18.751 11.322-17.403 22.281 1.245 10.13 9.864 17.554 19.816 17.554.813 0 1.637-.05 2.464-.151 7.156-.88 14.35-2.075 21.383-3.554 10.805-2.272 17.721-12.873 15.449-23.677-2.273-10.808-12.876-17.723-23.678-15.451z"
          fill="url(#a)"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      class=" animate-spin"
      height="512"
      viewBox="0 0 512 512"
      width="512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <lineargradient
        gradientTransform="matrix(.9996 0 0 -.9996 -.022 513.71)"
        gradientUnits="userSpaceOnUse"
        id="a"
        x1="0.625"
        x2="511.648"
        y1="257.824"
        y2="257.824"
      >
        <stop
          offset="0"
          stop-color="#00f2fe"
        />
        <stop
          offset="0.021"
          stop-color="#03effe"
        />
        <stop
          offset="0.293"
          stop-color="#24d2fe"
        />
        <stop
          offset="0.554"
          stop-color="#3cbdfe"
        />
        <stop
          offset="0.796"
          stop-color="#4ab0fe"
        />
        <stop
          offset="1"
          stop-color="#4facfe"
        />
      </lineargradient>
      <path
        d="M407.497 103.365c-40.873-40.873-95.217-63.383-153.02-63.383s-112.147 22.51-153.02 63.383l-9.585 9.585h65.66c11.041 0 19.991 8.95 19.991 19.991s-8.95 19.991-19.991 19.991l-71.701-.001c-15.86-.191-30.735-6.515-41.886-17.807-11.112-11.253-17.274-26.172-17.353-42.011V19.991C26.592 8.95 35.542 0 46.583 0s19.991 8.95 19.991 19.991v61.714l6.611-6.611C121.61 26.669 185.994 0 254.477 0S387.345 26.669 435.77 75.094c31.333 31.333 54.094 70.346 65.825 112.823 2.939 10.642-3.307 21.652-13.948 24.591-1.78.492-3.572.727-5.334.727-8.767.001-16.811-5.811-19.258-14.674-9.894-35.825-29.105-68.743-55.558-95.196zM127.759 431.233a218.234 218.234 0 01-14.207-11.531c-8.273-7.314-20.907-6.533-28.217 1.738-7.313 8.272-6.534 20.907 1.737 28.218a258.494 258.494 0 0016.821 13.652 19.901 19.901 0 0011.918 3.954c6.103 0 12.131-2.786 16.055-8.059 6.59-8.859 4.752-21.382-4.107-27.972zM23.096 297.764c11.002-.923 19.173-10.589 18.251-21.591a219.948 219.948 0 01-.763-18.289c0-11.041-8.95-19.991-19.991-19.991S.602 246.843.602 257.884c0 7.207.304 14.484.903 21.628.875 10.434 9.613 18.323 19.898 18.323.561 0 1.126-.023 1.693-.071zm36.514 48.879c-4.545-10.062-16.387-14.536-26.448-9.991-10.062 4.544-14.535 16.386-9.991 26.447a256.362 256.362 0 009.759 19.366c3.659 6.557 10.462 10.25 17.473 10.25 3.298 0 6.642-.818 9.726-2.539 9.641-5.382 13.093-17.559 7.711-27.2a215.6 215.6 0 01-8.23-16.333zM210.335 468.85a215.33 215.33 0 01-17.673-4.645c-10.548-3.258-21.742 2.649-25.002 13.199-3.26 10.549 2.65 21.742 13.198 25.002a256.192 256.192 0 0020.97 5.512 20.07 20.07 0 004.272.462c9.197 0 17.477-6.388 19.513-15.743 2.35-10.789-4.49-21.439-15.278-23.787zm282.882-209.234c-11.012-.995-20.72 7.102-21.72 18.098a216.88 216.88 0 01-2.425 18.124c-1.926 10.871 5.326 21.246 16.197 23.173 1.18.209 2.353.31 3.511.31 9.513 0 17.944-6.815 19.661-16.507a257.549 257.549 0 002.874-21.476c1.001-10.996-7.103-20.721-18.098-21.722zm-21.571 97.166c-9.601-5.451-21.803-2.083-27.254 7.518a217.28 217.28 0 01-9.69 15.504c-6.242 9.106-3.921 21.549 5.186 27.792a19.892 19.892 0 0011.285 3.503c6.37 0 12.633-3.04 16.507-8.69a257.432 257.432 0 0011.484-18.375c5.45-9.601 2.084-21.802-7.518-27.252zm-87.642 75.347a217.02 217.02 0 01-15.196 10.183c-9.425 5.752-12.4 18.055-6.648 27.479 3.768 6.172 10.344 9.579 17.082 9.579 3.55 0 7.145-.946 10.396-2.931a257.062 257.062 0 0018.008-12.067c8.903-6.528 10.829-19.039 4.301-27.943-6.529-8.901-19.037-10.83-27.943-4.3zm-82.853 37.039a216.53 216.53 0 01-18.031 2.998c-10.959 1.347-18.751 11.322-17.403 22.281 1.245 10.13 9.864 17.554 19.816 17.554.813 0 1.637-.05 2.464-.151 7.156-.88 14.35-2.075 21.383-3.554 10.805-2.272 17.721-12.873 15.449-23.677-2.273-10.808-12.876-17.723-23.678-15.451z"
        fill="url(#a)"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
